ansible/roles/retronas.role.samba.system.nolink/tasks/main.yaml

```diff
@@ -0,0 +1,108 @@
+- name: "{{ my_name }} - Include systems map"
+  ansible.builtin.include_vars: "retronas_systems.yml"
+
+- name: "{{ my_name }} - Load RetroNAS config"
+  ansible.builtin.include_vars: retronas_vars.yml
+
+- name: "{{ my_name }} - build top level"
+  ansible.builtin.file:
+    path: "{{ retronas_path }}/{{ system_key }}"
+    owner: "{{ retronas_user }}"
+    group: "{{ retronas_group }}"
+    state: directory
+    mode: "0775"
+
+- name: "{{ my_name }} - build top level share paths"
+  ansible.builtin.file:
+    path: "{{ retronas_path }}/{{ system_key }}/{{ item.name }}"
+    owner: "{{ retronas_user }}"
+    group: "{{ retronas_group }}"
+    state: directory
+    mode: "0775"
+  loop: "{{ top_level_paths }}"
+  when:
+    - top_level_paths is defined
+    - item.enabled is true
+
+- name: "{{ my_name }} - build layout list (set fact) (top level paths)"
+  ansible.builtin.set_fact:
+    path_list: "{{ path_list|default([]) + [ \"mkdir -p \" + retronas_path + '/' + system_key + '/' + item.1.name + '/' + item.0[system_key] + \";\" ] }}"
```

Comment:
This is one of the differences from the existing samba.system playbook. I use mkdir instead of ln -s to create the folders.